CHANGELOG.md

```diff
@@ -0,0 +1,169 @@
+- `linalg.slogdet`: add complex number support ([gh-567](https://github.com/data-apis/array-api/pull/567))
+- `linalg.solve`: add complex number support ([gh-566](https://github.com/data-apis/array-api/pull/566))
+- `linalg.svd`: add complex number support ([gh-561](https://github.com/data-apis/array-api/pull/561))
+- `linalg.svdvals`: add complex number support ([gh-562](https://github.com/data-apis/array-api/pull/562))
+- `linalg.trace`: add support for specifying output array data type ([gh-502](https://github.com/data-apis/array-api/pull/502))
+- `linalg.trace`: add complex number support ([gh-541](https://github.com/data-apis/array-api/pull/541))
+- `linalg.vector_norm`: add complex number support ([gh-550](https://github.com/data-apis/array-api/pull/550))
+
+* * *
+
+### Additions
+
+#### Data Types
+
+- `complex64`: single-precision complex floating-point numbers ([gh-418](https://github.com/data-apis/array-api/pull/418))
+- `complex128`: double-precision complex floating-point numbers ([gh-418](https://github.com/data-apis/array-api/pull/418))
+- Type promotion rules: real-complex and complex-complex data type promotion guidance ([gh-491](https://github.com/data-apis/array-api/pull/491))
+- Guidance for mixing arrays and Python `complex` scalars ([gh-513](https://github.com/data-apis/array-api/pull/513))
+- Guidance for data type variability across devices ([gh-515](https://github.com/data-apis/array-api/pull/515))
+- Guidance for complex number ordering ([gh-527](https://github.com/data-apis/array-api/pull/527))
+- Guidance for complex number equality ([gh-528](https://github.com/data-apis/array-api/pull/528))
+
+**note**: conforming implementations must define a default complex floating-point data type.
+
+#### APIs
+
+- `__array_api_version__`: string representing the version of the array API specification ([gh-480](https://github.com/data-apis/array-api/pull/480))
+- `__complex__`: convert a zero-dimensional array to a Python `complex` object ([gh-497](https://github.com/data-apis/array-api/pull/497))
+- `conj`: return the complex conjugate of a complex number ([gh-446](https://github.com/data-apis/array-api/pull/446))
+- `finfo.dtype`: floating-point data type ([gh-485](https://github.com/data-apis/array-api/pull/485))
+- `iinfo.dtype`: integer data type ([gh-485](https://github.com/data-apis/array-api/pull/485))
+- `imag`: return the imaginary component of a complex number ([gh-496](https://github.com/data-apis/array-api/pull/496))
+- `isdtype`: test whether a provided `dtype` is of a specified data type kind ([gh-503](https://github.com/data-apis/array-api/pull/503))
+- `real`: return the real component of a complex number ([gh-427](https://github.com/data-apis/array-api/pull/427))
+- `take`: return elements of an array along a specified axis ([gh-416](https://github.com/data-apis/array-api/pull/416))
+
+#### Extensions
+
+- `fft`: Fast Fourier Transforms (FFT) ([gh-189](https://github.com/data-apis/array-api/pull/189))
```

Comment:
"One new extension was added, the `fft` namespace for Fast Fourier Transforms (FFT)"